packages/terra-button-group/package.json

```diff
@@ -27,13 +27,13 @@
     "react": "^15.4.2",
     "react-dom": "^15.4.2",
     "terra-base": "^2.11.0",
-    "terra-button": "^x.0"
+    "terra-button": "^1.14.0"
   },
   "dependencies": {
     "classnames": "^2.2.5",
     "prop-types": "^15.5.8",
     "terra-base": "^2.11.0",
-    "terra-button": "^x.0"
+    "terra-button": "^1.14.0"
```

Comment:
Will lerna bump this when it publishes?

Reply:
Lerna will prompt if the package should be bumped when publishing and it will update all the other packages that depend on it. I had to bump it up in https://github.com/cerner/terra-core/pull/1155 because no all packages have been uplifted. Now that all are, it should be safe to revert the version bump.